app/(beforeLogin)/(auth)/signup/component/Select.tsx

```
@@ -0,0 +1,75 @@
+
+  const isAllChecked = termsList.every((t) => checks.get(t.termId));
+  const isAllRequiredChecked = termsList
+    .filter((t) => t.isRequired)
+    .every((t) => checks.get(t.termId));
+
+  if (isLoading) return <div>약관을 불러오는 중...</div>;
+  if (isError) return <div>{error.message}</div>;
+
+  const handleAllCheck = (checked: boolean) => {
+    const newMap = new Map();
+    termsList.forEach((term) => newMap.set(term.termId, checked));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)